setup.py

```diff
@@ -57,10 +57,10 @@ def parse_requirements(filename):
     packages=find_packages(),
     include_package_data=True,
     package_data={
-      'libcloudforensics': ['libcloudforensics/scripts/*']
+        'libcloudforensics': ['libcloudforensics/scripts/*']
```

Comment:
Since we're renaming the CLI tool, shouldn't we rename the parent module to `cloudforensics` as well? This would also make it a little more consistent with the name of the github repo

Reply:
This probably requires more work and is probably worth its own PR, as well as extra notifications for projects that depend on LCF.
Tracking in #229

Reply:
Sounds good, thanks